adapter/internal/oasparser/config_generator.go

```diff
@@ -268,7 +268,7 @@ func castPoliciesToEnforcerPolicies(policies []model.Policy) []*api.Policy {
 			}
 		}
 		enforcerPolicies[i] = &api.Policy{
-			PolicyName: policy.PolicyName,
+			PolicyName: policy.PolicyName, //TODO (renuka): remove this not requited name and version
 			Action:     policy.Action,
 			Order:      uint32(policy.Order),
```

Comment:
didn't apim remove the order attrib from the policy? if it's still there, what is the usage of it, did we handle it in cc?

Reply:
yes order is not supported and should be removed, already have an issue for this https://github.com/wso2/product-microgateway/issues/2660

for enforcer side we also do not need policy name or policy version, that should also removed